src/main/resources/templates/projects.ftl

```diff
@@ -1,38 +1,43 @@
 <#--@ftlvariable name="projects" type="java.util.List"-->
 <#-- @ftlvariable name="projects" type="com.web.link.checker.project.model.Project"-->
-<!DOCTYPE html>
-<meta charset="utf-8">
-<html>
-
-<head>
-    <title>Project Page</title>
-</head>
-
-<body>
-
+<#import "layout/defaultLayout.ftl" as layout>
+<@layout.defaultLayaout "Projects">
     <h1>Projects</h1>
-    <table border="1" cellpadding="5" cellspacing="0" width="200" height="75">
+    <div id="formResponse">
+        <#if projectProjectionPage.getContent()?has_content>
+        <#else>
+            <div class="alert alert-warning">
+                <span class='label-remove-project'> You do not have any projects</span>
+            </div>
+        </#if>
+    </div>
+    <p>
+        <a href="/projects/save" class="btn btn-primary">Add</a>
+    </p>
+    <table class="table table-striped">
         <thead>
         <tr>
             <th>Name</th>
             <th>Actions</th>
         </tr>
         </thead>
         <tbody>
-        <#list projectProjections as projectProjection>
-        <tr>
-            <td>${projectProjection.name}</td>
-            <td>
-              <a href="/projects/${projectProjection.uuid}/delete">Delete</a>
-              <a href="/projects/${projectProjection.uuid}/update">Edit</a>
-            </td>
-        </tr>
-        </#list>
+            <#list projectProjectionPage.getContent() as projectProjection>
+            <tr>
+                <td>${projectProjection.name}</td>
+                <td>
+                    <a href="/projects/${projectProjection.uuid}/update">
+                        <i class="glyphicon glyphicon-pencil" aria-hidden="true">&nbsp;</i>
```

Comment:
aria-hidden="true" тоже хз что такое

Reply:
Modern versions of assistive technologies will announce CSS generated content, as well as specific Unicode characters. To avoid unintended and confusing output in screen readers (particularly when icons are used purely for decoration), we hide them with the aria-hidden="true" attribute.(c)